Review 2024.4 changelog and documentation changes (#17030)
nvaccess/nvda: user_docs/en/userGuide.md

### System Requirements {#SystemRequirements}

#### Minimum System Requirements {#MinimumSystemRequirements}

* Operating Systems: all 32-bit and 64-bit editions of Windows 8.1, Windows 10, Windows 11, and all Server Operating Systems starting from Windows Server 2012 R2.
  * both AMD64 and ARM64 variants of Windows are supported.
  * Note 32-bit operating systems are no longer under active support.

>   * Note that Windows 8.1 and Windows Server versions older than 2022 are no longer under active support.

Reviewer: please restore these lines.

>   * Note 32-bit operating systems are no longer under active support.
>   * Note that Windows 8.1 and Windows Server versions older than 2022 are no longer under active support.

Author: ```suggestion
  * Note 32-bit operating systems are no longer under active support.
  * Note that Windows 8.1 and Windows Server versions older than 2022 are no longer under active support.
```

### System Requirements {#SystemRequirements}

#### Minimum System Requirements {#MinimumSystemRequirements}

* Operating Systems: all 32-bit and 64-bit editions of Windows 8.1, Windows 10, Windows 11, and all Server Operating Systems starting from Windows Server 2012 R2.
  * both AMD64 and ARM64 variants of Windows are supported.

>   * Note 32-bit operating systems are no longer under active support.

Author: I don't think this is really worth having in the user guide. @Qchristensen, what do you think? If we end up keeping it, I agree that we should note that it is Microsoft that doesn't support 32 bit Windows any more.

>   * Note 32-bit operating systems are no longer under active support.

Reviewer 2: I agree - we state that Windows 8.1 is supported - but don't note that it reached end of support (by Microsoft) over a year ago.  We do of course encourage users to be running supported version of their operating system and other software, but in the user guide here, we really only need to note what WE support.   I'm in favour of dropping that line entirely.

>   * Note 32-bit operating systems are no longer under active support.

Author: ```suggestion
```

>   * Note 32-bit operating systems are no longer under active support.

Reviewer: This is meant to be referring that we as NV Access are no longer supporting or testing with these version. Please see #16666

>   * Note that Windows 8.1 and Windows Server versions older than 2022 are no longer under active support.

Author: ```suggestion
```

This can be especially helpful in applications used to read large pieces of text, like structured documents or books.
In such documents, knowing where paragraphs start may be useful to understand the structure of the content, or to set bookmarks or annotations based on paragraph position.

> The options include using double-spaces for softer paragraph breaks, and the paragraph symbol, pilcrow (¶), for more obvious paragraph breaks.

Author: I actually think this sentence could do with a bit more work. I'm not fond of repeating "paragraph breaks" multiple times in one sentence. And I think we can simplify "double-spaces" to "two spaces". I've also changed "soft" to "subtle".

```suggestion
The options include using two spaces as a subtle paragraph break, and the paragraph symbol, pilcrow (¶), as a more obvious one.
```

What do you think, @XLTechie, @CyrilleB79, @Qchristensen?

Reviewer 2: Do we need to explain why you might use each?  What about:
```suggestion
The options include using double-spaces or the paragraph symbol, pilcrow (¶).
```

Reviewer 3: I prefer Sasha's proposal, i.e. do not drop the use case for each type of paragraph marker. Indeed, describing the use cases in the user guide can help the user make their choice.

Describing the use cases in the user guide is not necessary when the use case is obvious from the option. But in the present description, I feel that the use case description is a plus.

Reviewer 3: Also cc @nvdaes, author of this change.

Reviewer 4: I also prefer Sascha's proposal. I for one would not have known why I should choose one over the other, if it hadn't been written with this moderate explanation.

> The options include using double space for softer paragraph breaks, or alternatively the paragraph symbol, pilcrow (¶), for more obvious paragraph breaks.

Reviewer 3: I agree with @XLTechie's proposal. But in addition:
1. I'd not separate the two possibilities by a ";" (semicolon). The two options should remaine in the same phrase IMO. At least that would be more grammatically correct in French (my native language).
2. Also, I'd use "and" rather than "or" since both possibilities are in the options list, not one or the other.

As usual, for English fixes from myself, feel free to accept or decline.

```suggestion
The options include using double spaces, for softer paragraph breaks, and the paragraph symbol, pilcrow (¶), for more obvious paragraph breaks.
```

> The options include using double space for softer paragraph breaks, or alternatively the paragraph symbol, pilcrow (¶), for more obvious paragraph breaks.

Author: Agree with dropping "alternatively", and changing "double space" to "double-spaces". I also agree with switching "or" to "and" here, as it's one or the other.
```suggestion
The options include using double-spaces for softer paragraph breaks, and the paragraph symbol, pilcrow (¶), for more obvious paragraph breaks.
```

* Off: NVDA will not report these font attributes.
* Speech: NVDA will announce when these font attributes change.
* Braille: NVDA will display bold, italic, and underlines in braille.

> Note that your chosen braille table must support displaying these attributes.

Author: It has been updated to be consistent with the features shipped this version. Is there a reason to remove it? I think it adds context on the options.

> Note that your chosen braille table must support displaying these attributes.

Reviewer: Sorry this comment wasn't clear and the original point I was getting to also don't make sense
